demos/object_detection_demo/python/models.lst

```diff
@@ -55,3 +55,5 @@ yolo-v3-tf
 yolo-v3-tiny-tf
 # For --architecture_type=yolov4
 yolo-v4-tf
+# For --architecture_type=yolov4-tiny
+yolo-v4-tiny-tf
```

Comment:
is there are possibility to not introduce a new category/arch_type for just tiny variant of yolo-v4?

Reply:
I wish it was possible but the IR does not contain `[yolo]` layer parameters.

Reply:
Changed model arch detection by name property of the IR. That's weak but the only way.